### ISSUE TITLE
 The configuration in the panel does not change

### ISSUE THREAD

**Issue by kronxg** [UnamSanctam/SilentCryptoMiner#2165]
I changed the configuration in the panel, nothing changes. What is the problem? I change it in Pastebin, they still have the same thing, even the new ones

**Comment by UnamSanctam** [comment]
I also answered it in your other issue. Is the web panel configuration or "Remote Configuration" correct? It will only change it if the pool configuration is valid.

**Comment by kronxg** [comment]
I recorded proxies in the pool. Well, in general the panel seems to be installed normally, the miners are coming

**Comment by UnamSanctam** [comment]
Can you post your configuration that you're attempting to use (you can censor things like the wallet address)? Both the web panel configuration and the "Remote Configuration" are guaranteed to be working.

**Comment by kronxg** [comment]
спрятал*

**Comment by UnamSanctam** [comment]
And you can connect to your proxy using a normal xmrig (https://github.com/xmrig/xmrig)? You also don't need a wallet or password for an xmrig-proxy (wallet will show as the worker name in the proxy). And is SSL enabled on the proxy port that you're trying to connect to?

**Comment by kronxg** [comment]
Yes, miners are installed, 40 miners. port - 1123, from here - https://zephyr.herominers.com/#how-to-mine-zephyr-zeph
I don’t know if ssl is installed, please check on the website.

**Comment by kronxg** [comment]
How to select a configuration, save it, but the configuration does not change

**Comment by kronxg** [comment]
I change the configuration in the panel for all the miners that are there. In the old panel 1.7.0 everything was fine. Doesn't change in 1.8.0.
![dddfff](https://github.com/UnamSanctam/SilentCryptoMiner/assets/159646191/8446e9a5-60c5-404e-937e-bf5a94c22e90)
Here, instead of a pool, there should be an ip proxy

**Comment by UnamSanctam** [comment]
You should try with another real pool in the configuration to see if the problem is with your proxy.

**Comment by UnamSanctam** [comment]
Hmm, I just tested it and it seems to work fine for me in 1.8.0. Are you doing anything special?

**Comment by kronxg** [comment]
I have different wallets in my miners right now, I want to change the configuration for all of them, but geez, it doesn't change(

**Comment by kronxg** [comment]
I have 2 domains, 1.7.0 checked today, everything is fine there, changed to 1.8.0 - no change. It turns out everything was fine, as soon as I switched to 1.8.0 everything broke. Maybe it has something to do with connecting to you? You have the database stored?

**Comment by UnamSanctam** [comment]
Hmm, I have asked others and it's working for them too (and the beta testers also reported it working). Can you try clearing your cache and then re-save the configuration? Since I can see in the screenshot that it's using an old cached version of the styling.

**Comment by kronxg** [comment]
They say they need to issue some kind of access rights

**Comment by UnamSanctam** [comment]
Well the database file and database folder needs to have write permissions (so that the database can be written to), though since miners can connect for you then they already are writable.

**Comment by kronxg** [comment]
On file

755

The first time you run it, it's possible that it won't start, then 777

After that, you will need to revert to 755.

**Comment by UnamSanctam** [comment]
Possibly, maybe in case it's unable to write the WAL log (for better performance and to reduce the change of corruption). Though with the correct owner that shouldn't be the case, at least whenever I've helped others install it, but if you do need it then you might be able to do 770 instead of 777.

**Comment by kronxg** [comment]
I'll check it out and let you know!

**Comment by kronxg** [comment]
![ваввввв](https://github.com/UnamSanctam/SilentCryptoMiner/assets/159646191/bc19bb2a-3343-46de-ae08-e74a8186a5d1)
the default rights were:

**Comment by Silentsniper1** [comment]
> It's not about the proxy, it's about the panel. Check for yourself, the configuration does not change, I restored the old panel 1.7.0, I had 2k miners there, updated to the new one - 1.8.0, now 160 miners, the configuration does not change here, although the configuration changed in version 1.7.0. Fix it! ![dfgfdggd](https://private-user-images.githubusercontent.com/159646191/304436143-ac8c9474-5bfc-4533-8e93-367224e6f5c5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5ODc1NjIsIm5iZiI6MTcwODk4NzI2MiwicGF0aCI6Ii8xNTk2NDYxOTEvMzA0NDM2MTQzLWFjOGM5NDc0LTViZmMtNDUzMy04ZTkzLTM2NzIyNGU2ZjVjNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyNlQyMjQxMDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04M2QwMDlmMWQ2ZGU4ZWU0YjYzMzdkZDZlZjczMDBjYjAxNWJlOTJlYmE0ZTVhNTM2ZTJiODYzMjVkNDUwNzVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Y9uQBxcbQHwNNEe5SRqIV6YArhGKW91pAb108jiDaoo)

bro how the hell did you manage to get 2K miners can i work for you or something?
